config/crd/bases/cellular.config.nephio.org_amfconfigs.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.13.0
+  name: amfconfigs.cellular.config.nephio.org
+spec:
+  group: cellular.config.nephio.org
+  names:
+    kind: AmfConfig
+    listKind: AmfConfigList
+    plural: amfconfigs
+    singular: amfconfig
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            description: 'APIVersion defines the versioned schema of this representation
+              of an object. Servers should convert recognized schemas to the latest
+              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+            type: string
+          kind:
+            description: 'Kind is a string value representing the REST resource this
+              object represents. Servers may infer this from the endpoint the client
+              submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+            type: string
+          metadata:
+            type: object
+          spec:
+            description: AmfConfigSpec defines the characteristics of a deployment
+              of a network function
+            properties:
+              amfName:
+                description: amfName defines name of the amf NF
+                maxLength: 253
+                minLength: 1
+                type: string
+              servedGuami:
+                description: ServedGuami defines information related to GUAMI
+                items:
+                  description: ServedGuami information related to GUAMI
+                  properties:
+                    amfId:
+                      description: AmfId defines the AMF Identifier
+                      properties:
+                        amfPointer:
+                          description: amfPointer uniquely identifies the AMF in AMF
+                            set
+                          maxLength: 6
```

Comment:
Does the lenght need to be restricted for these?
Is this from the 3pp spec?

Reply:
Yes @efiacor I took from yang models and specs the limit all the limits